releasenotes/notes/add-mx-plugin.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: release-notes/v2
+kind: feature
+area: telemetry
+issue:
+  - 28410
+releaseNotes:
+  - |
+    **Added** HTTP and TCP metadata exchange telemetry filter configuration plugin.
```

Comment:
What is the upgrade strategy? Currently if you upgrade from 1.8, the old proxies will get 2x mx filters (from envoyfilter and from the new code)

Reply:
- I think I will revert the 1.9 yaml change to make this PR purely focusing on adding http/tcp mx plugins and ensure insertion order is correct (aka take mx filter as precedence over other filters).
- And then doing a separate PR for handling version kind of switching to trigger this plugin.

That will make things clear and easy to review.

Reply:
That sounds good. We should remove this release notes as well since nothing should really be enabled by this PR.